### PR TITLE
Added PageNotFound component

### PR DESCRIPTION
**Pull request by shibangi23**
### Functionality:
Added PageNotFound component

### Solution:
Created multiple files which implement page not found component

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
Please go to the route `localhost:4200/any` to check the component

### CODE REVIEW

**Comment by shibangi23** [comment]
Made some changes in the UI for 404 error display . Please review it.

File: src/app/pagenotfound/pagenotfound.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		</div>
+		
+		<div class="contant_box_404">
+		<h3 class="h2">
+		Looks like you're lost
+		</h3>
+		
+		<p>The page you are looking for is not avaible!</p>
+		
+		<a href="" class="link_404">Go to Home</a>
```
Comment (viv9k) [inline src/app/pagenotfound/pagenotfound.component.html:19]: Is this working

**Review by viv9k — CHANGES_REQUESTED** [review]
Please fix it

File: src/app/pagenotfound/pagenotfound.component.html (hunk 1 above)
Comment (shibangi23) [reply src/app/pagenotfound/pagenotfound.component.html:19]: Yes it's working.

**Review by viv9k — COMMENTED** [review]
Need demo on this.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**